### PR TITLE
(maint) Release 0.26.0

### PR DESCRIPTION
**Pull request by jpogran**

### CODE REVIEW

**Comment by glennsarti** [comment]
Want to add in the facts endpoint?

**Comment by jpogran** [comment]
The facts endpoint PR updated the changelog for it

**Comment by jpogran** [comment]
All issues from 0.26.0 are added, and version is set

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,13 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 ### Added
 
 - ([GH-242](https://github.com/puppetlabs/puppet-editor-services/issues/242)) Add Puppet Facts Endpoint
+- ([GH-245](https://github.com/puppetlabs/puppet-editor-services/issues/245)) Use object cache for Puppet Facts Endpoint
```
Comment (glennsarti) [inline CHANGELOG.md:12]: This is in the wrong section. Should be 0.26.0 not unreleased

**Pull request merged**